core/src/epoch_slots.rs

```diff
@@ -132,7 +141,7 @@ impl Uncompressed {
             if *s < self.first_slot {
                 return i;
             }
-            if *s - self.first_slot >= self.slots.capacity() {
+            if *s - self.first_slot >= self.slots.len() {
```

Comment:
`Uncompressed::new()` will always create a bitvec with capacity equal to len, but it's much more correct to use len here